sqldelight-gradle-plugin/src/test/fixtures/dots-in-folder/src/main/sqldelight/com.sample.test/Test.sq

```diff
@@ -0,0 +1,3 @@
+CREATE TABLE test (
```

Comment:
This doesn't seem like it's something we should support. Java doesn't care because the folder structure is meaningless, it's just a convention. But for `.sq` files the folders are the only source of package name.

What do you think about erroring if the folder name doesn't match what's allowed for a single package name?